pkg/builder/controller.go

```diff
@@ -241,6 +250,10 @@ func (blder *Builder) doController(r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
-	blder.ctrl, err = newController(name, blder.mgr, controller.Options{Reconciler: r})
+	ctrlOptions := controller.Options{
+		Reconciler:              r,
+		MaxConcurrentReconciles: blder.ctrlOptions.MaxConcurrentReconciles,
```

Comment:
probably should invert this so that we don't have to manually copy over fields

Reply:
namely:

```go
ctrlOptions := builder.ctrlOptions
ctrlOptions.Reconciler = r
```

Reply:
Done.